op_crates/web/02_abort_signal.js

```diff
@@ -36,11 +36,20 @@
         throw new TypeError("Illegal constructor.");
       }
       super();
-      this.onabort = null;
-      this.addEventListener("abort", (evt) => {
-        const { onabort } = this;
-        if (typeof onabort === "function") {
-          onabort.call(this, evt);
+      // Dom specification section 8.1.5.1
```

Comment:
Is this the right section?  I can't seem to find it in the DOM Spec.  Points to something about XPath.

Reply:
https://html.spec.whatwg.org/multipage/webappapis.html#event-handler-attributes this is the HTML specification and not the DOM specification (they link to each other) - I will update the comment.